app/views/layouts/_tl_options.html.haml

```diff
@@ -3,7 +3,8 @@
 #tl_options_div
   %form#form_div{:name           => 'angularForm',
                  'ng-controller' => 'timelineOptionsController',
-                 'ng-show'       => 'afterGet',
+                 'ng-show'       => 'vm.afterGet',
+                 'miq-form'      => true,
```

Comment:
`miq-form` by itself is useless if you don't also provide the `model` and `model-copy` attributes here ;)